### PR TITLE
docs: Add missing space

### PR DESCRIPTION
**Pull request by bbodenmiller**
**Does this change impact docs?**
- [x] Yes, PR includes docs updates
- [ ] Yes, issue opened: # 
- [ ] No

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

**Review by mbevc1 — DISMISSED** [review]

File: website/content/en/v0.27.0/getting-started/getting-started-with-eksctl/_index.md [hunk 1]
```diff
@@ -123,7 +123,7 @@ Karpenter should terminate the now empty nodes.
 
 This section describes optional ways to configure Karpenter to enhance its capabilities.
 In particular, the following commands deploy a Prometheus and Grafana stack that is suitable for this guide but does not include persistent storage or other configurations that would be necessary for monitoring a production deployment of Karpenter.
-This deployment includes two Karpenter dashboards that are automatically onboarded to Grafana.They provide a variety of visualization examples on Karpenter metrics.
+This deployment includes two Karpenter dashboards that are automatically onboarded to Grafana. They provide a variety of visualization examples on Karpenter metrics.
```
Comment (jonathan-innis) [inline website/content/en/v0.27.0/getting-started/getting-started-with-eksctl/_index.md:126]: Thanks for this change! Can you propagate this change throughout all versions where it has occurred?

Comment (bbodenmiller) [reply website/content/en/v0.27.0/getting-started/getting-started-with-eksctl/_index.md:126]: Updated.

**Comment by mbevc1** [comment]
There is one in `website/content/en/preview/getting-started/getting-started-with-karpenter` as well, which is going to be part on next release.

**Comment by bbodenmiller** [comment]
> There is one in `website/content/en/preview/getting-started/getting-started-with-karpenter` as well, which is going to be part on next release.

Updated

**Review by bwagner5 — APPROVED** [review]
LGTM

**Pull request merged**